omod/src/main/java/org/openmrs/module/idgen/web/controller/IdgenRestController.java

```diff
@@ -9,11 +17,11 @@
 @RequestMapping("/rest/" + RestConstants.VERSION_1 + IdgenRestController.IDGEN_NAMESPACE)
 public class IdgenRestController extends MainResourceController {
 
- public static final String IDGEN_NAMESPACE = "/idgen";
```

Comment:
Is changing this class part of the current ticket?

Reply:
yes. I was removing the spaces.